package.json

```diff
@@ -6,6 +6,7 @@
   "engines": {
     "node": ">= 16.0.0"
   },
+  "sideEffects": false,
```

Comment:
I read the webpack manual for optimization, and I get the impression this flag helps with reducing the size of the output.  Should other terriajs-packages (terriajs-server, thredds-catalog-crawler, etc.) have a `package.json` that contains `sideEffects: false` as well?